python/pyiceberg/catalog/__init__.py

```diff
@@ -25,6 +26,41 @@
 from pyiceberg.table.partitioning import UNPARTITIONED_PARTITION_SPEC, PartitionSpec
 from pyiceberg.table.sorting import UNSORTED_SORT_ORDER, SortOrder
 from pyiceberg.typedef import EMPTY_DICT, Identifier, Properties
+from pyiceberg.utils.config import Config, merge_config
+
+logger = logging.getLogger(__name__)
+
+_env_config = Config()
+
+
+def load_catalog(name: str, uri: str, **properties: str | None) -> Catalog:
+    from pyiceberg.catalog.rest import RestCatalog
+
+    supported_catalogs: dict[str, type[Catalog]] = {"http": RestCatalog}
```

Comment:
Can we create this once and make it a constant?

Reply:
Unfortunately not, because that would introduce a circular import, therefore we have to import it in the method itself.

Reply:
Makes sense!